frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -49,102 +57,101 @@ const StyledTruncator = styled(Truncator)(({ theme }) => ({
 }));
 
 const StyledHeader = styled('div', {
-    shouldForwardProp: (prop) => prop !== 'draggable' && prop !== 'disabled',
-})<{ draggable: boolean; disabled: boolean }>(
-    ({ theme, draggable, disabled }) => ({
-        padding: theme.spacing(0.5, 2),
-        display: 'flex',
-        gap: theme.spacing(1),
-        alignItems: 'center',
-        paddingLeft: draggable ? theme.spacing(1) : theme.spacing(2),
-        color: disabled
-            ? theme.palette.text.secondary
-            : theme.palette.text.primary,
-    }),
-);
+    shouldForwardProp: (prop) => prop !== 'disabled',
+})<{ draggable: boolean; disabled: boolean }>(({ theme, disabled }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    color: disabled ? theme.palette.text.secondary : theme.palette.text.primary,
+}));
+
+const StyledHeaderInner = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    gap: theme.spacing(1),
+}));
```

Comment:
Allow for the draggable item with an offset and use a nested div for header and headerItemsLeft (badges etc)